src/extension/settingsHelper.ts

```diff
@@ -67,4 +67,11 @@ export class SettingsHelper {
                 return settingsJson["typescript.tsdk"] || "";
             });
     }
+
+    public static read(name: string): Q.Promise<any> {
```

Comment:
It's not... My first implementation of this feature read the configuration from settings.json. I left this method because I think it's likely we'll use it in the future.